services/QuillLMS/client/app/bundles/Teacher/components/unit_template_categories/unit_template_category.tsx

```diff
@@ -0,0 +1,101 @@
+import * as React from 'react'
```

Comment:
I think this is copied from somewhere, but you might want to consider using ES6 syntax for this component since we are trying to move to that for new React files!